README.md

```diff
@@ -3,3 +3,10 @@
 | Route | Description | Parameters | Expected Output |
 |-------|-------------|------------|-----------------|
 | `POST /api/login`| logs the user in | body: `{ email: string, password: string}` | `{ token: string, refresh_token: string }` |
+| `POST /api/sign-in` | Sign in a user | body `{ firstName: string, lastName: string, email: string, password: string}` | `{ token: string, refresh_token: string}`|
+| `POST /api/user/task` | Create a List | body `{ title: string, description: string}` | `{ title: string, description: string}` |
+| `GET /api/user/task` | List all task | body `{title: string, description: string}` | `{title: string, description: string}` |
+| `GET /api/user/task/{id}` | List specific task | body `{title: string, description: string}` | `{title: string, description: string}` |
+| `PUT /api/user/task/{id}/edit` | Edit specific task | body `{title: string, description: string}` | `{title: string, description: string}` |
+| `PUT /api/user/task/{id}/done` | Realized specific task | body `{title: string, description: string}` | `{title: string, description: string}` |
+| `DELETE /api/user/task/{id}/delete` | Delete specific task | body `{title: string, description: string}` | `{title: null, description: null}` |
```

Comment:
you don't need the last part of these paths.
PUT already implies that you are editing, and you don't need 2 paths to edit the same entity, you use the same path changing only the body content.

Also DELETE http method already implies deletion